client/src/renderer/components/Clipboard/index.jsx

```diff
@@ -0,0 +1,66 @@
+/** @format */
+
+import "./styles.scss";
+import { Component, h } from 'preact';
+import clipboard from "electron-clipboard-extended";
+
+export default class Clipboard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { clipboardStack: [] };
+  }
+
+  componentDidMount() {
+    clipboard.startWatching();
+    clipboard.on("text-changed", () => {
+      this.setState(prevState => {
+        const text = clipboard.readText();
+        return {
+          clipboardStack: [
+            ...prevState.clipboardStack,
+            new ClipboardItem("txt", text),
```

Comment:
You can safely inline this variable
```suggestion
            new ClipboardItem("txt", clipboard.readText()),
```